sentinel.go

```diff
@@ -509,8 +529,20 @@ func (c *sentinelFailover) slaveAddrs(ctx context.Context) ([]string, error) {
 			return addrs, nil
 		}
 		_ = c.closeSentinel()
+	} else {
+		// shuffle the sentinelAddrs to make the load more balanced
+		rand.Shuffle(len(c.sentinelAddrs), func(i, j int) {
```

Comment:
Should this code really be in `else` clause?

Reply:
> Should this code really be in `else` clause?

I add `QuerySentinelRandomly` option to decide whether to shuffle.

Reply:
I am not sure this is an essential option so I probably going to remove it. Let me know if you think it **is** essential.

I guess your original code was correct and we only want to shuffle this list once.